core/contracts/financial_templates/Liquidatable.sol

```diff
@@ -387,6 +386,5 @@ contract Liquidatable is PricelessPositionManager {
         returns (LiquidationData storage liquidation)
     {
         liquidation = liquidations[sponsor][uuid];
```

Comment:
Looking at the test now (beginning line 214), it seems that this line will throw. Any point in wrapping it in a require statement so we can make error messages more clear?

Reply:
Will add a require here.

Reply:
Done.